content/en/apps/guides/performance/purging.md

```diff
@@ -105,7 +105,9 @@ This function takes four parameters:
 - `userCtx`, an object with the user's `roles` as fields. For more information read the [documentation for the User Context Object](https://docs.couchdb.org/en/stable/json-structure.html#userctx-object).
 - `contact`, the contact document of a patient or other contact who has reports about them.
 - `reports`, an array of all reports for that subject that are present on the server.
-- `messages`, an array of sms messages that the contact has sent or received
+- `messages`, an array of sms messages that the contact has sent or received.
+- `chtScriptApi` (Optional) the CHT API that provides CHT-Core Framwework's functions to other parts of the app. More info on the API can be found [here]({{<ref "apps/reference/_partial_cht_api" >}}).
+- `persmissionSettings` (Optional) string or array of permissions that the user has. More info on permissions can be found [here]({{<ref "apps/reference/app-settings/user-permissions" >}}).
```

Comment:
Please add something like "added in 4.3.0" to these two paramters so it's clear.